### PR TITLE
Fix remaining apps management issues

### PR DESCRIPTION
**Pull request by juliusknorr**
Fixes the rest of #9767

- Asynchronously fetch groups to fix invalid request #9767
- Don't open sidebar when enabling/disabling an app from the list #9767
- Hide apps after uninstalling #9767
- Fix error display when enabling an app fails
- Add transition to fix "Enabling app should animate the entry away (collapse), otherwise it's not visible what happens" from #10094

### CODE REVIEW

File: settings/src/components/appManagement.vue [hunk 1]
```diff
@@ -54,6 +54,9 @@
 			}
 		},
 		methods: {
+			asyncFindGroup(query) {
+				return this.$store.dispatch('getGroups', {search: query, limit: 5, offset: 0});
```
Comment (skjnldsv) [inline settings/src/components/appManagement.vue]: So we only show the 5 first matches?
I'm fine with it!

**Review by skjnldsv — APPROVED** [review]
Awesome!!

File: settings/src/components/appManagement.vue (hunk 1 above)
Comment (juliushaertl) [reply settings/src/components/appManagement.vue]: Yes, it will fetch more results while typing.

**Review by MorrisJobke — APPROVED** [review]
Tested and works 👍

**Pull request merged**